src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background: no-repeat url(images/star-active.svg);
+  background-position: 50% 50%;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: no-repeat url(images/star.svg);
+  background-position: 50% 50%;
+  margin-right: 4px;
```

Comment:
There shouldn't be margins after last start
<img width="1440" alt="Screenshot 2023-02-22 at 19 38 58" src="https://user-images.githubusercontent.com/52370890/220710802-b9551192-860a-4936-a0b5-e23fb2e9936b.png">

Reply:
Done
Added .stars__star:last-child